madsim-tonic-build/src/prost.rs

```diff
@@ -674,7 +674,7 @@ impl Builder {
 
         // generate origin
         config.out_dir(out_dir);
-        builder.compile_with_config(config, protos, includes)?;
+        builder.compile_protos_with_config(config, protos, includes)?;
```

Comment:
CI's clippy check failed without this.

Reply:
I cannot compile locally with this change. Weird... 🤣

Reply:
Might be something to do with the tool chain version. I recall I had issues until I bumped my tool chain to latest.